### PR TITLE
http: add GetVersion API

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve?

Issue Number: ref #7300

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

File: client/http/interface.go [hunk 1]
```diff
@@ -723,3 +724,16 @@ func (c *client) GetMicroServiceMembers(ctx context.Context, service string) ([]
 	}
 	return members, nil
 }
+
+// GetVersion gets the release version of the PD binary.
+func (c *client) GetVersion(ctx context.Context) (string, error) {
```
Comment (JmPotato) [inline client/http/interface.go:729]: What about using a more specified name like `GetPDVersion`?

**Comment by lance6716** [comment]
ptal

File: client/http/interface.go [hunk 2]
```diff
@@ -723,3 +724,16 @@ func (c *client) GetMicroServiceMembers(ctx context.Context, service string) ([]
 	}
 	return members, nil
 }
+
+// GetVersion gets the release version of the PD binary.
```
Comment (HuSharp) [inline client/http/interface.go:728]:
```suggestion
// GetPDVersion gets the release version of the PD binary.
```

**Review by JmPotato — APPROVED** [review]

**Comment by HuSharp** [comment]
please sign DCO

**Review by HuSharp — APPROVED** [review]

**Comment by lance6716** [comment]
please help merge this PR

**Pull request merged**